### DISCUSSION TITLE
ELI5 Compress Superheat and Subcool temperautre meaning

### DISCUSSION THREAD

**Discussion by starsoccer** [ccutrer/waterfurnace_aurora#54] [Q&A]
New to geothermal systems in general and trying to understand what superheat and subcool temps reflect exactly. Currently on my unit they are both are 10 degrees

**Comment by mdallaire** [comment]
That is one thing I like about this module, the vast amount of information available about my system. However just like you I have no idea of what most of it means...

**Reply by starsoccer** [reply]
Agreed thats why I started this discussion in hopes that someone with more knowledge would be able to provide this information
